pipelines/plot_and_save_state_forecast.R

```diff
@@ -20,24 +20,58 @@ save_forecast_figures <- function(model_run_dir,
     pyrenew_model_name,
     timeseries_model_name
   )
-
+  transform_to_name <- c("identity" = "", "log10" = "_log")
+  timescale_to_dat_prefix <- c(
+    "daily" = "",
+    "epiweekly" = "epiweekly_",
+    "epiweekly_other" = "epiweekly_"
+  )
+  timescale_to_ci_name <- c(
+    "daily" = "forecast_ci",
+    "epiweekly" = "epiweekly_forecast_ci",
+    "epiweekly_other" =
+      "forecast_with_epiweekly_other_ci"
+  )
   figure_save_tbl <-
     expand_grid(
       target_disease = unique(processed_forecast$combined_dat$disease),
-      y_transform = c("identity", "log10")
+      y_transform = c("identity", "log10"),
+      timescale = c("daily", "epiweekly", "epiweekly_other"),
+    ) |>
+    filter(!(target_disease == "Disease" &
+      timescale == "epiweekly_other")) |>
+    mutate(
+      transform_name = transform_to_name[y_transform],
+      dat_prefix = timescale_to_dat_prefix[timescale],
+    ) |>
+    mutate(
+      figure_path = path(
+        model_run_dir,
+        glue(paste0(
```

Comment:
```suggestion
        glue(
```
the paste is not required. Make sure to delete the extra closing paren.